### ISSUE TITLE
DatePicker - No proper state is announced for the current day

### ISSUE THREAD

**Issue by yanbxice** [microsoft/fluentui#15453]
### Environment Information

- **Package version(s)**: @uifabric/date-time@v7.17.1     office-ui-fabric-react@v7.134.1
- **Browser and OS versions**: Edge

### Describe the issue:

When screen reader users navigate to the present day, visually it is displayed using the background color but the same information is not conveyed to the screen reader users.

This is the same issue with [#14611](https://github.com/microsoft/fluentui/issues/14611), but is was closed. I have checked with the new version of both packages, I can still repro. Rather than upgrade the package versions, any other code change needed from my side?

#### Actual behavior:

screen reader reads the current date the same as the other dates.

#### Expected behavior:

screen reader should convey the information to users that it is the current day.

### If applicable, please provide a codepen repro:

https://codepen.io/yanbxice/pen/BajwqWo

**Comment by paulgildea** [comment]
@yanbxice Thanks for filing an issue and doing some initial investigation. We really appreciate it.

@tomi-msft / @lorejoh12 Can you coordinate with @yanbxice to see if there's something we can to do help in the short term?

**Comment by paulgildea** [comment]
@smhigley FYI

**Comment by JustSlone** [comment]
This is a duplicate of #14898 which was fixed by #14806. aria-current="date" was added. This is working in Safari/VoiceOver Chrome/NVDA, but aria-current is not supported by Edge/Narrator, so it cannot work.
